cmd/theatre-envconsul/acceptance/acceptance.go

```diff
@@ -29,19 +35,6 @@ metadata:
 spec:
   serviceAccountName: secret-reader
   restartPolicy: Never
-  initContainers:
```

Comment:
We no longer require this init container as the work to provision vault is done within the prepare block